pkgs/desktops/gnome/extensions/update-extensions.py

```diff
@@ -22,6 +22,7 @@
     "44": "44",
```

Comment:
Shouldn't we drop some old versions at some point?

Reply:
PRs welcomed (I am not familiar with `pkgs/desktops/gnome/extensions`).